internal_api.go

```diff
@@ -105,11 +129,18 @@ func (e *Enforcer) removePolicy(sec string, ptype string, rule []string) (bool,
 
 // removePolicies removes rules from the current policy.
 func (e *Enforcer) removePolicies(sec string, ptype string, rules [][]string) (bool, error) {
-	rulesRemoved := e.model.RemovePolicies(sec, ptype, rules)
+	rulesRemoved, effects := e.model.RemovePolicies(sec, ptype, rules)
```

Comment:
can `effects` be `nil` here when `no policy` found to be removed

Reply:
Returns `nil` will convert zero slice, it's working.